src/main/steps/task-list/template.njk

```diff
@@ -0,0 +1,367 @@
+    {% extends "common/page.njk" %}
+        {% block content %}
+            <div class="govuk-grid-column-full"></div>
+            <div class="govuk-grid-row">
+
+                <div class="govuk-grid-column-two-thirds">
+                    <h1 class="govuk-heading-xl">
+                        {{title}}
+                    </h1>
+
+                    <ol class="app-task-list">
+                        <!-- **************************************** 1. Add application details **************************************** -->
+                        <li>
+                            <h2 class="app-task-list__section">
+                                <span class="app-task-list__section-number">1. </span>{{section1}}
+                            </h2>
+                                        <ul class="app-task-list__items">
+                <div class="govuk-inset-text home-office-inset-text">
+                    <p>{{insetTextLine1}}</p>
+                    <ul class="govuk-list govuk-list--bullet">
+                        <li>{{insetTextLine2}}</li>
+                        <li>{{insetTextLine3}}</li>
+                    </ul>
+                    <p>{{insetTextLine4}}</p>
+                    <ul class="govuk-list govuk-list--bullet">
+                        <li>{{insetTextLine5}}</li>
+                        <li>{{insetTextLine6}}</li>
+                        <li>{{insetTextLine7}}</li>
+                        <li>{{insetTextLine8}}</li>
+                    </ul>
+                </div>
+
+                                <!-- Number of applicants -->
+                                <li class="app-task-list__item">
+                                    <span class="app-task-list__task-name">
+                                        <a href="/applying-with" class="govuk-link">
+                                            {{section1details1}}
+                                        </a>
+                                    </span>
+
+                                    {% if (data['number-of-applicants']) %}
```

Comment:
this condition in incorrect, we need to pass a flag to indicate whether this particular section has been answered